src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let caunt = 0;
+
+  return function(a) {
+    return function(b) {
+      caunt++;
+
+      if (caunt > 3) {
+        return caunt % 2 === 0 ? 'Bzzz... Error!' : a + b;
+      }
+
+      return a + b;
+    };
+  };
 }
```

Comment:
First of all - typo in word count `caunt -> count`/
no need ternary - you can conmbine condition

```suggestion
  let count = 0;

  return function(a) {
    return function(b) {
      count++;

      if (count > 3 && count % 2 === 0) {
        return 'Bzzz... Error!';
      }

      return a + b;
    };
  };
}
```

Reply:
Fixed thanks nice.